server/server.go

```diff
@@ -194,6 +195,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&cfg.LogFormat, "log.format", log.LogfmtFormat, "Output log messages in the given format. Valid formats: [logfmt, json]")
 	cfg.LogLevel.RegisterFlags(f)
 	f.BoolVar(&cfg.LogSourceIPs, "server.log-source-ips-enabled", false, "Optionally log the source IPs.")
+	f.BoolVar(&cfg.LogSourceIPsFull, "server.log-source-ips-full", false, "Log all source hosts instead of only the first one. Only used if server.log-source-ips-enabled is true")
```

Comment:
Can you clarify this message a bit? e.g.
```suggestion
	f.BoolVar(&cfg.LogSourceIPsFull, "server.log-source-ips-full", false, "Log all source IPs instead of only the originating one. Only used if server.log-source-ips-enabled is true")
```

Reply:
Change made